lib/mnemosyne/middleware/rack.rb

```diff
@@ -109,7 +109,10 @@ def scan_response(trace, response)
 
         return unless headers
 
-        trace.meta[:content_type] = headers['Content-Type']
+        trace.meta.merge!({
+          content_type: headers['Content-Type'],
+          location: headers['Location'],
```

Comment:
@jgraichen Will mnemosyne-server just start showing this in a span's meta information? Currently, it does not seem to include all the meta fields added in this middleware...

Reply:
No, it wont.